notebooks/basic_example.ipynb

```diff
@@ -35,68 +48,123 @@
     "from tq42.compute import HardwareProto"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "1eb308d8",
+   "metadata": {},
+   "source": [
+    "## 1. Create a client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "f57f6788",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"device_code\":\"Srug_gMfZZfJ8dugONS9xkE3\",\"user_code\":\"RGPB-FGXZ\",\"verification_uri\":\"https://auth.terraquantum.io/activate\",\"expires_in\":900,\"interval\":5,\"verification_uri_complete\":\"https://auth.terraquantum.io/activate?user_code=RGPB-FGXZ\"}\n",
```

Comment:
please clean your outputs before committing a notebook. You are exposing secrets (even if they are expired :D).

Reply:
Absolutely – that was an oversight. Thanks for the reminder :D